FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.SchemaManagement/ContainerRegistry/ContainerRegistryTemplateProvider.cs

```diff
@@ -64,23 +70,27 @@ public class ContainerRegistryTemplateProvider : IContainerRegistryTemplateProvi
             }
             catch (ContainerRegistryAuthenticationException authEx)
             {
-                _logger.LogError(authEx, "Failed to access container registry.");
+                _diagnosticLogger.LogError("Failed to access container registry.");
+                _logger.LogInformation(authEx, "Failed to access container registry.");
                 throw new ContainerRegistrySchemaException("Failed to access container registry.", authEx);
             }
             catch (ImageFetchException fetchEx)
             {
-                _logger.LogError(fetchEx, "Failed to fetch template image.");
+                _diagnosticLogger.LogError("Failed to fetch template image.");
```

Comment:
Seems this error message is more converter user oriented. Shall we add some custom terms related to our pipeline?

Reply:
We can refine content in other PR to make it more customer friendly.